staging/src/k8s.io/apiserver/pkg/server/filters/priority-and-fairness.go

```diff
@@ -90,12 +100,16 @@ func WithPriorityAndFairness(
 		}
 
 		var classification *PriorityAndFairnessClassification
-		note := func(fs *flowcontrol.FlowSchema, pl *flowcontrol.PriorityLevelConfiguration) {
+		note := func(fs *flowcontrol.FlowSchema, pl *flowcontrol.PriorityLevelConfiguration, flowDistinguisher string) {
 			classification = &PriorityAndFairnessClassification{
 				FlowSchemaName:    fs.Name,
 				FlowSchemaUID:     fs.UID,
 				PriorityLevelName: pl.Name,
 				PriorityLevelUID:  pl.UID}
+
+			httplog.AddKeyValue(ctx, "apf_pl", truncateLogField(pl.Name))
+			httplog.AddKeyValue(ctx, "apf_fs", truncateLogField(fs.Name))
+			httplog.AddKeyValue(ctx, "apf_d", truncateLogField(flowDistinguisher))
```

Comment:
Do we need to log flowDistinguisher too?

Reply:
There was a discussion above and I think the conclusion was that it may be useful. @lavalamp @MikeSpreitzer

Reply:
fs and pl names seem to be a good starting point for now, maybe we can add `flowDistinguisher` in the future if we need to?

Reply:
I frankly am not sure how much value there is in including the flow distinguisher.  It _is_ *definitely* relevant.  My concern is whether it is difficult enough to discover from other evidence that it is worth putting in this log line that is printed for every request.  I will not object either way.

If we _do_ include it, I suggest a slightly longer name: `apf_fd` (for "flow distinguisher", a logical companion to "flow schema").